packages/server/src/lib/fetchTickerData.ts

```diff
@@ -2,8 +2,11 @@ import { TickerData, TickerSettings } from '@sdc/shared/types';
 import { Response } from 'node-fetch';
 import fetch from 'node-fetch';
 import { buildReloader, ValueProvider } from '../utils/valueReloader';
+import { logError } from '../utils/logging';
+import { TickerName } from '@sdc/shared/dist/types';
 
-const tickerUrl = 'https://support.theguardian.com/ticker.json';
+const tickerUrl = (name: TickerName): string =>
+    `https://support.theguardian.com/ticker/${name}.json`;
```

Comment:
this requires a small fastly change - we can route any requests to `/ticker/*` to the bucket